lib/DDG/Spice/NobelPrize.pm

```diff
@@ -0,0 +1,43 @@
+package DDG::Spice::NobelPrize;
+
+# ABSTRACT: Write an abstract here
+
+# Start at http://docs.duckduckhack.com/walkthroughs/forum-lookup.html if you are new
+# to instant answer development
+
+use DDG::Spice;
+use Data::Dumper;
+
+spice is_cached => 1;
+spice proxy_cache_valid => "200 1d"; # defaults to this automatically
+
+spice wrap_jsonp_callback => 1;
+
+spice to => 'http://api.nobelprize.org/v1/prize.json?' 
+                . '&year=$1' 
+                . '&category=$2';
+
+
+# Triggers - https://duck.co/duckduckhack/spice_triggers
+triggers startend => 'nobel';
```

Comment:
We need to reign this in a lot. We already know that we can expect the queries to look like:

- nobel prize winners {year}
- nobel {category} prize winners
- nobel {category} prize winners {year}

I would make the triggers "nobel prize" +"nobel peace prize", "nobel physics prize" etc. (you can build the latter if you put the categories into an array. You can `map` over them to build the strings and also `join` them to build your regex 👍